src/main/java/mate/academy/dao/MovieSessionDao.java

```diff
@@ -6,9 +6,13 @@
 import mate.academy.model.MovieSession;
 
 public interface MovieSessionDao {
-    List<MovieSession> findAvailableSessions(Long movieId, LocalDate date);
+    MovieSession add(MovieSession movieSession);
 
     Optional<MovieSession> get(Long id);
 
-    MovieSession add(MovieSession session);
+    List<MovieSession> findAvailableSessions(Long movieId, LocalDate date);
+
+    boolean update(MovieSession movieSession);
```

Comment:
Consider changing the return type of the `update` method to `void` or `MovieSession` to better reflect the operation's outcome. Returning a `boolean` might not be necessary if the method is expected to always succeed or throw an exception otherwise.